lib/components/NatAppBarSearch/NatAppBarSearch.test.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+
+import { NatAppBarSearch } from './NatAppBarSearch';
+
+describe('Header component', () => {
+  it('Should render container component correctly', () => {
+    const header = renderer.create(
+      <NatAppBarSearch
+        title="Title test"
+        onPress={jest.fn()}
+      />,
+    ).toJSON();
+
+    expect(header).toMatchSnapshot();
```

Comment:
Name the snapshot, e.g.,
`expect(header).toMatchSnapshot('NatAppBarSearch default render');`